src/main/java/com/graphhopper/converter/ConverterApplication.java

```diff
@@ -13,60 +15,67 @@
 
 import javax.servlet.DispatcherType;
 import javax.ws.rs.client.Client;
+
 import java.util.EnumSet;
 
 /**
- * @author Robin Boldt
+ * @author Robin Boldt,David Masclet
  */
 public class ConverterApplication extends Application<ConverterConfiguration> {
 
-    public static void main(String[] args) throws Exception {
-        new ConverterApplication().run(args);
-    }
-
-
-    @Override
-    public String getName() {
-        return "graphhopper-nominatim-converter";
-    }
-
-    @Override
-    public void initialize(Bootstrap<ConverterConfiguration> bootstrap) {
-        // nothing to do yet
-    }
-
-    @Override
-    public void run(ConverterConfiguration converterConfiguration, Environment environment) throws Exception {
-
-        JerseyClientConfiguration cfg = converterConfiguration.getJerseyClientConfiguration();
-        cfg.setTimeout(Duration.seconds(10));
-        cfg.setConnectionTimeout(Duration.seconds(10));
-        cfg.setConnectionRequestTimeout(Duration.seconds(10));
-
-        final Client client = new JerseyClientBuilder(environment).using(cfg)
-                .build(getName());
-
-        if (converterConfiguration.isNominatim()) {
-            final ConverterResourceNominatim resource = new ConverterResourceNominatim(
-                    converterConfiguration.getNominatimUrl(),
-                    converterConfiguration.getNominatimReverseUrl(),
-                    converterConfiguration.getNominatimEmail(),
-                    client);
-            environment.jersey().register(resource);
-        }
-
-        if (converterConfiguration.isOpenCageData()) {
-            final ConverterResourceOpenCageData resource = new ConverterResourceOpenCageData(
-                    converterConfiguration.getOpenCageDataUrl(), converterConfiguration.getOpenCageDataKey(), client);
-            environment.jersey().register(resource);
-        }
-
-        if (converterConfiguration.isHealthCheck()) {
-            final NominatimHealthCheck healthCheck =
-                    new NominatimHealthCheck(converterConfiguration.getNominatimUrl(), client);
-            environment.healthChecks().register("template", healthCheck);
-        }
-
-        environment.servlets().addFilter("ip-filter", new IPFilter(converterConfiguration.getIPWhiteList(), converterConfiguration.getIPBlackList())).addMappingForUrlPatterns(EnumSet.allOf(DispatcherType.class), true, "/*");
-    }
+	public static void main(String[] args) throws Exception {
+		new ConverterApplication().run(args);
+	}
+
+
```

Comment:
Would you mind to check the formatting? Sometimes there are 2 or 3 lines between methods. Is there a reason for this or can we just use one empty line?

Reply:
Also there is a change for this whole file, probably because of tabs vs spaces. It would be great if we could keep the changeset as small as possible.

Reply:
Please do it separately. I use Eclipse and appreciate if you can do it in another commit. thanks

Reply:
I done it but not sure it is correct